.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2.1
+
+jobs:
+  push:
+    machine: true
+    environment:
+      - GOPATH: /home/circleci/go
+      - LOCAL_FILE_PATH: /home/
+      - GOOGLE_CLOUD_PROJECT: Stackrox Infra
+      - GOOGLE_APPLICATION_CREDENTIALS: /home/StackRoxInfra-980b6fe329cb.json
+    working_directory: /home/circleci/go/src/github.com/stackrox/k8s-istio-cve-pusher
+    steps:
+      - run:
+          name: Copy configs
+          command: cp configs/StackRoxInfra-980b6fe329cb.json /home/
+      - run:
+          name: Install dependencies
+          command: go get ./...
+      - run:
+          name: Build
+          command: |
+            go build
+      - run:
+          name: Run tests
+          command: go test -v ./main -run Test
+      - run:
+          name: Run job
+          command: ./main/main
+
+workflows:
+  version: 2
+  cron:
+    jobs:
+      - push
+    triggers:
+      - schedule:
+          cron: "0 * * * *"
+          filters:
```

Comment:
Is this really a valid spec? How would a cronjob trigger make sense with a branch filter?
Tip: you can install the circleci CLI (`brew install circleci`) and then run `circleci config validate` from the project root to validate the config.